client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java

```diff
@@ -87,6 +101,21 @@ public void f(Element element) {
             }
         });
 
+        $(cartButton).click(new Function() {
+            @Override
+            public void f() {
+                if (shoppingBagOpen) {
+                    $(shoppingBagWidget).show();
+                    $(cartIcon).attr("class", "iconClose");
```

Comment:
isn't the `iconXXX` style in a Resource file? If so you should reference that instead.